packages/balancer/services/autonomous_fund/service.yaml

```diff
@@ -22,6 +22,8 @@ models:
       allowed_lp_addresses: ${ALLOWED_LP_ADDRESSES:list:["0x87cc0d34f6111c8A7A4Bdf758a9a715A3675f941"]}
       enforce_allowlist: ${ENFORCE_ALLOWLIST:bool:true}
       observation_interval: ${OBSERVATION_INTERVAL:int:600}
+      service_registry_address: '0x48b6af7B12C71f09e2fC8aF4855De4Ff54e775cA'
+      on_chain_service_id: ${ON_CHAIN_SERVICE_ID:int:2}
       consensus:
```

Comment:
why not `share_tm_config_on_startup` too? Let's bring this to parity with el coll imo

Reply:
Yes, makes sense.